ui/summary.go

```diff
@@ -157,34 +176,26 @@ func StrWidth(s string) (n int) {
 	return
 }
 
-// SummaryData represents data passed to Summarize.
-type SummaryData struct {
-	Opts    lib.Options
-	Root    *lib.Group
-	Metrics map[string]*stats.Metric
-	Time    time.Duration
-}
```

Comment:
I thought this struct was superfluous, and opted for passing required args to `Write()` instead.

Reply:
OK, I'll revert this.

I'm just not a fan of using structs as args, since it hides the complexity, as a lot of args are usually an indication that the function should be broken down / simplified. Which is certainly the case here, but a major refactor can probably wait.

Reply:
Resolved in dd29bd39.